src/app/payment/views/IntegratedCheckoutView/CheckoutViewWrapper.tsx

```diff
@@ -82,10 +80,9 @@ export interface CheckoutViewManager {
 }
 
 const ONE_YEAR_IN_MONTHS = 12;
-
 const IS_PRODUCTION = envService.isProduction();
-
 const RETURN_URL_DOMAIN = IS_PRODUCTION ? process.env.REACT_APP_HOSTNAME : 'http://localhost:3000';
+const STATUS_CODE_ERROR = [409, 422];
```

Comment:
Perhaps it would be better to create an object, so that the key that identifies the reason for the status code, I think, would be clearer. 
Then you can extract the values from the object and use the list for comparison

Reply:
I agree 👍🏽